best-practices/README.md

```diff
@@ -161,6 +161,8 @@ Postgres
 * Consider a [partial index] for queries on booleans.
 * Constrain most columns as [`NOT NULL`].
 * [Index foreign keys].
+* Use an `ORDER BY` clause on queries where the results will be displayed to a
+  user, as queries without one may return results in changing, arbitrary order.
```

Comment:
This feels like it needs an article: `... queries without one may return results in a changing, arbitrary order`.